server/data.js

```diff
@@ -65,55 +65,55 @@ const data = [
     //10 sponser results
     {   id: 101,
         url: "https://www.cat.com/",
-        title: "Cat.com | Cat | Caterpillar",
+        title: "(!SPONSORS!) Cat.com | Cat | Caterpillar",
```

Comment:
Sponsors look really  cool and work, but can't see the "(!SPONSER)" Title in final push.